### PR TITLE
Update section purger plugin to sectionbundled

### PR DESCRIPTION
**Pull request by MarkCalvert**

### CODE REVIEW

**Review by steveworley — CHANGES_REQUESTED** [review]
Great! Update hook looks good, I think we will also need to update config/install so when a new site installs baywatch they will have the bundled purger configured directly.

**Review by steveworley — APPROVED** [review]

**Comment by steveworley** [comment]
Hey @GROwen would you mind having a review of this proposed change? There is a [bundled purger](https://git.drupalcode.org/project/section_purge/-/blob/master/src/Plugin/Purge/Purger/SectionBundledPurger.php) plugin in the Section module which provides the ability to invalidate multiple tags in one request.

This is an update hook to update the section purger configuration values for deployed sites (as they're not all using the config provided by Baywatch).

File: baywatch.install [hunk 1]
```diff
@@ -588,3 +588,46 @@ function baywatch_update_8032() {
...
+  $channels = $purge_logger_channels_config->get('channels');
+  if (is_array($channels)) {
+    foreach ($channels as &$channel) {
+      // Only update channel id for section purger
+      if (isset($channel['id']) && strpos($channel['id'], "purger_section_") === 0) {
+        // Update the id to section bundled
+        // channel id is in the format of purger_section_{purger_id}
+        $parts = explode('_', $channel['id']);
+        $purger_id = end($parts);
+        $channel['id'] = "purger_sectionbundled_$purger_id";
```
Comment (GROwen) [inline baywatch.install:626]: Should this be 
```suggestion
        $channel['id'] = "purger_sectionbundled_{$purger_id}";
```

**Comment by GROwen** [comment]
Hey @steveworley and @MarkCalvert Is the sectionbundled purger an existing purger or do we need to bump the version of the module?

Also, this belongs in https://github.com/dpc-sdp/bay_platform_dependencies/ We moved platform dependencies there a little while ago.

**Comment by steveworley** [comment]
Thanks @GROwen — sorry I forgot that platform deps were split out. Nope, the section module was bundled with this for a while — unsure why we didn't use this in the initial set up of the module.

Will get it moved to the right place, thanks!

**Comment by MarkCalvert** [comment]
Migrated PR to https://github.com/dpc-sdp/bay_platform_dependencies/pull/2

**Pull request closed without merging**